.conda/meta.yaml

```diff
@@ -0,0 +1,100 @@
+###############################################################################
+## File for Anaconda.org
+## WARNING : strings PYPI_VERSION, PYPI_URL and PYPI_SHA256 will be replaced
+## by CI script get_pypi_info.py, leave them as is.
+###############################################################################
+
+{% set name = "OpenFisca-Core" %}
+{% set version = "PYPI_VERSION" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: PYPI_URL
+  sha256: PYPI_SHA256
+
+build:
+  noarch: python
+  number: 4
```

Comment:
What's the meaning of `number` and `4` here? 😊

Reply:
`number: 4` is a sub-version to allow publishing the same PyPi version multiple time.
It has to be 0 when we switch to a new version.
I will do it.